package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "govuk_design_system",
+  "version": "0.1.0",
+  "private": true,
+  "description": "GOV.UK Design System",
+  "scripts": {
```

Comment:
Likewise with this – are we required to specify it if it's not going to be used?

Reply:
Yes, we can get rid of the scripts section, since we're not going to do anything fancy with npm on this repo. At least for the moment.